docs/HACKING.md

```diff
@@ -13,22 +13,30 @@ The operator image needs to be pushed to a remote repository:
 podman push quay.io/<insert username>/wmco:$VERSION_TAG
 ```
 ## Development workflow
-To run the operator on a cluster (for testing purpose only) or to run the e2e tests for WMCO against an OpenShift 
-cluster set up, we need to setup the following environment variables.
+With the Operator image built and pushed to a remote repository, the operator can be deployed either directly,
+or through OLM.
+
+For each deployment method a kubeconfig for an OpenShift or OKD cluster must be exported
 ```shell script
 export KUBECONFIG=<path to kubeconfig>
+```
 
-# SSH key to be used for creation of cloud-private-key secret 
-export KUBE_SSH_KEY_PATH=<path to RSA type ssh key>
-
-# on AWS only:
-export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
+### Deploying the operator directly
+```shell script
+make deploy IMG=<pushed_image>
```

Comment:
Rather than `<pushed_image>`, `quay.io/<insert username>/wmco:$VERSION_TAG` might make it easier to understand which image we are talking about here.